dependencies.yaml

```diff
@@ -231,7 +231,7 @@ dependencies:
       match: REVISION:\ '\d+'
 
   - name: "registry.k8s.io/build-image/kube-cross (v1.25-go1.20)"
-    version: v1.25.0-go1.20.10-bullseye.0
+    version: v1.25.0-go1.20.11-bullseye.0
```

Comment:
Should we change this given 1.25 has reached EOL?

Reply:
oh yes, i thought is this month, but actually is eol :) will update

Reply:
done